setup.py

```diff
@@ -62,8 +62,11 @@ def get_version(path):
             "metpy",
             "numpy",
             "quantnn>=0.0.5",
-            "torch",
+            "torch==1.13.0",
```

Comment:
OK
A side note, in our conda env YAML file we have
```
  - torchvision==0.14.1
  - torchaudio==0.13.1
```
I assume this isn't critical at all.

Reply:
I tried installing it yesterday and it seems that torch<=2.0 is required somehow.